jormungandr/src/blockchain/process.rs

```diff
@@ -70,11 +70,13 @@ pub fn handle_input(
                 });
 
             if let Some(msg_box) = explorer_msg_box {
+                /*
                 msg_box
                     .try_send(ExplorerMsg::NewBlock(new_block_ref))
                     .unwrap_or_else(|err| {
                         error!(info.logger(), "cannot add block to explorer: {}", err)
                     });
+                */
```

Comment:
Will this be re-enabled in a follow-up change?

Reply:
yes, thanks, I need to talk with @enzoc4 about it.